CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Adapted resolveImports to handle AstImport objects from parseImports: PR [#966](https://github.com/tact-lang/tact/pull/966)
```

Comment:
```suggestion
- The `parseImports` function now returns AST import nodes instead of raw strings: PR [#966](https://github.com/tact-lang/tact/pull/966)
```